ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/DefaultCommandValues.java

```diff
@@ -10,13 +10,8 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package tech.pegasys.ethsigner.core.config;
+package tech.pegasys.ethsigner;
```

Comment:
nit: Git does some funny stuff - but that renames is a bit odd :/

Reply:
It's a new class. I don't know why github thinking it's a package switch.